src/app/courses/credits-course/course-credits.component.html

```diff
@@ -0,0 +1 @@
+<h1>Course credits of {{courses}}</h1>
```

Comment:
{{courses}} is it for all courses or individual course,
Please follow style guide for naming convention

Reply:
@razu9861 Updated. See https://github.com/ole-vi/planet/pull/259/commits/9d0e4a478ece1dfc7ee0ff4881b637947ded8b37